besu t8n fails on cancun

```
Request: test_mineBlocks
Alloc:
{
    "0x095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
        "balance" : "0x0de0b6b3a7640000",
        "code" : "0x600160010160005500",
        "nonce" : "0x00",
        "storage" : {
        }
    },
    "0x2adc25665018aa1fe0e6bc666dac8fc2697ff9ba" : {
        "balance" : "0x00",
        "code" : "0x",
        "nonce" : "0x01",
        "storage" : {
        }
    },
    "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
        "balance" : "0x0de0b6b3a7640000",
        "code" : "0x",
        "nonce" : "0x00",
        "storage" : {
        }
    }
}
Txs:
"0xf865f863800a83061a8094095e7baea6a6c7c4c2dfeb977efac326af552d87830186a0801ba0ffb600e63115a7362e7811894a91d8ba4330e526f22121c994c4692035dfdfd5a06198379fcac8de3dbfac48b165df4bf88e2088f294b61efb9a65fe2281c76e16"
{
    "data" : "0x",
    "gasLimit" : "0x061a80",
    "gasPrice" : "0x0a",
    "nonce" : "0x00",
    "to" : "0x095e7baea6a6c7c4c2dfeb977efac326af552d87",
    "value" : "0x0186a0",
    "v" : "0x1b",
    "r" : "0xffb600e63115a7362e7811894a91d8ba4330e526f22121c994c4692035dfdfd5",
    "s" : "0x6198379fcac8de3dbfac48b165df4bf88e2088f294b61efb9a65fe2281c76e16"
}
Env:
{
    "currentCoinbase" : "0x2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
    "currentNumber" : "0x01",
    "currentTimestamp" : "0x03e8",
    "currentGasLimit" : "0xff112233445566",
    "previousHash" : "0x533ceeb5465d31e67e95e07a1aee28f5631ac2f38aa4de1886046f9aa26f650a",
    "currentBeaconRoot" : "0x0000000000000000000000000000000000000000000000000000000000000000",
    "parentTimestamp" : "0x00",
    "parentDifficulty" : "0x00",
    "parentUncleHash" : "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
    "currentRandom" : "0x0000000000000000000000000000000000000000000000000000000000020000",
    "withdrawals" : [
    ],
    "parentBaseFee" : "0x0b",
    "parentGasUsed" : "0x00",
    "parentGasLimit" : "0xff112233445566",
    "parentBeaconBlockRoot" : "0x0000000000000000000000000000000000000000000000000000000000000000",
    "parentExcessBlobGas" : "0x00",
    "parentBlobGasUsed" : "0x00",
    "blockHashes" : {
        "0" : "0x533ceeb5465d31e67e95e07a1aee28f5631ac2f38aa4de1886046f9aa26f650a"
    }
}
/home/ubuntu/.retesteth/besu/start.sh --state.fork Cancun --state.reward 0 --state.chainid 1 --input.alloc /dev/shm/02fe4af4-0fb4-44f7-92c4-8eb1cb7bbf37/alloc.json --input.txs /dev/shm/02fe4af4-0fb4-44f7-92c4-8eb1cb7bbf37/txs.rlp --input.env /dev/shm/02fe4af4-0fb4-44f7-92c4-8eb1cb7bbf37/env.json --output.basedir /dev/shm/02fe4af4-0fb4-44f7-92c4-8eb1cb7bbf37 --output.result out.json --output.alloc outAlloc.json --output.errorlog /dev/shm/02fe4af4-0fb4-44f7-92c4-8eb1cb7bbf37/error.json

Res:

RAlloc:

Tool log: 
java.lang.IllegalArgumentException: Cannot construct instance of `org.apache.tuweni.bytes.Bytes32` (no Creators, like default constructor, exist): abstract types either need to be mapped to concrete types, have custom deserializer, or contain additional type information
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: org.hyperledger.besu.ethereum.referencetests.ReferenceTestEnv["parentBeaconBlockRoot"])
	at com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:4449)
	at com.fasterxml.jackson.databind.ObjectMapper.convertValue(ObjectMapper.java:4380)
	at org.hyperledger.besu.evmtool.T8nSubCommand.run(T8nSubCommand.java:208)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2026)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
	at picocli.CommandLine.execute(CommandLine.java:2170)
	at org.hyperledger.besu.evmtool.EvmToolCommand.execute(EvmToolCommand.java:277)
	at org.hyperledger.besu.evmtool.EvmToolCommand.execute(EvmToolCommand.java:243)
	at org.hyperledger.besu.evmtool.EvmTool.main(EvmTool.java:23)
Caused by: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Cannot construct instance of `org.apache.tuweni.bytes.Bytes32` (no Creators, like default constructor, exist): abstract types either need to be mapped to concrete types, have custom deserializer, or contain additional type information
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: org.hyperledger.besu.ethereum.referencetests.ReferenceTestEnv["parentBeaconBlockRoot"])
	at com.fasterxml.jackson.databind.exc.InvalidDefinitionException.from(InvalidDefinitionException.java:67)
	at com.fasterxml.jackson.databind.DeserializationContext.reportBadDefinition(DeserializationContext.java:1909)
	at com.fasterxml.jackson.databind.DatabindContext.reportBadDefinition(DatabindContext.java:408)
	at com.fasterxml.jackson.databind.DeserializationContext.handleMissingInstantiator(DeserializationContext.java:1354)
	at com.fasterxml.jackson.databind.deser.AbstractDeserializer.deserialize(AbstractDeserializer.java:274)
	at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:542)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeWithErrorWrapping(BeanDeserializer.java:564)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeUsingPropertyBased(BeanDeserializer.java:471)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1405)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:352)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:185)
	at com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:4444)
	... 13 more

```

--- a/ethereum/referencetests/src/main/java/org/hyperledger/besu/ethereum/referencetests/ReferenceTestEnv.java
+++ b/ethereum/referencetests/src/main/java/org/hyperledger/besu/ethereum/referencetests/ReferenceTestEnv.java
@@ -97,7 +97,7 @@ public class ReferenceTestEnv extends BlockHeader {
    */
   @JsonCreator
   public ReferenceTestEnv(
-      @JsonProperty("beaconRoot") final String beaconRoot,
+      @JsonProperty("parentBeaconBlockRoot") final String beaconRoot,
       @JsonProperty("blockHashes") final Map<String, String> blockHashes,
       @JsonProperty("ommers") final List<String> _ommers,
       @JsonProperty("previousHash") final String previousHash,

